Specs/DateTime/Dutch/DateTimePeriodParser.json

```diff
@@ -1669,15 +1866,42 @@
         "Start": 14,
         "Length": 55
       }
-    ],
-    "NotSupported": "dotNet",
-    "NotSupportedByDesign": "javascript,python,java"
+    ]
+  },
+  {
+    "Input": "Kan je ons op 9 december tussen 8 uur ‘s ochtends en 14 uur ‘s middags in plannen?",
+    "Context": {
+      "ReferenceDateTime": "2017-11-09T16:12:00"
+    },
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "9 december tussen 8 uur ‘s ochtends en 14 uur ‘s middags",
+        "Type": "datetimerange",
+        "Value": {
+          "Timex": "(XXXX-12-09T08,XXXX-12-09T14,PT6H)",
+          "FutureResolution": {
+            "startDateTime": "2017-12-09 08:00:00",
+            "endDateTime": "2017-12-09 14:00:00"
+          },
+          "PastResolution": {
+            "startDateTime": "2016-12-09 08:00:00",
+            "endDateTime": "2016-12-09 14:00:00"
+          }
+        },
+        "Start": 14,
+        "Length": 56
+      }
+    ]
   },
   {
     "Input": "Hi Cortana, plan een skype meeting in met Jennifer. Ik heb een afspraak van 30 minuten op vrijdagmiddag aanstaande nodig. ",
     "Context": {
       "ReferenceDateTime": "2017-11-13T16:12:00"
     },
+    "Comment": "Words that are related to each other, such as 'aanstaande' and 'vrijdag', may not be seperated by other words from a different entity such as 'middag'",
```

Comment:
You mean that in the current code, the modifier "next/upcoming" needs to be consecutive to the day of the week? Yeah... That's a limitation.

Reply:
yes exactly, the case with "aanstaande vrijdagmiddag" works fine.